pom.xml

```diff
@@ -95,6 +95,9 @@
         <scala_2.11.version>2.11.12</scala_2.11.version>
         <scala_2.12.version>2.12.10</scala_2.12.version>
 
+        <!-- Unsupported connectors -->
+        <exlude.unsupported>None</exlude.unsupported>
```

Comment:
We have a conditional ignore tag for this purpose. It's much easier to use. https://github.com/AbsaOSS/spline/issues/606#issuecomment-613321957